contracts/StablecoinConverter.sol

```diff
@@ -206,11 +197,9 @@ contract StablecoinConverter is EpochTokenLocker {
         }
         // doing all subtractions after all additions (in order to avoid negative values)
         for (uint i = 0; i < owners.length; i++) {
-            uint128 executedBuyAmount = volumes[i];
-            uint128 executedSellAmount = getExecutedSellAmount(
-                executedBuyAmount,
-                currentPrices[orders[owners[i]][orderIds[i]].buyToken],
-                currentPrices[orders[owners[i]][orderIds[i]].sellToken]
+            (, uint128 executedSellAmount) = getTradedAmounts(
+                volumes[i],
```

Comment:
nit: I prefer the term `amounts` here

Reply:
I think volumes is very common. No one talks about trading amounts, it is always trading volume :)

But you are right, that we are inconsistent with volumes and amounts. Might tackle it in another PR